pkg/domain/infra/tunnel/containers.go

```diff
@@ -138,3 +139,24 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 	}
 	return reports, nil
 }
+
+func (ic *ContainerEngine) ContainerTop(ctx context.Context, options entities.TopOptions) (*entities.StringSliceReport, error) {
+	if options.Latest {
+		last := 1
+		ctrs, err := containers.List(ic.ClientCxt, nil, nil, &last, nil, nil, nil)
+		if err != nil {
+			return nil, err
+		}
+		if len(ctrs) != 1 {
+			return nil, errors.Errorf("error looking up latest container: expected 1 but got %d", len(ctrs))
+		}
+		options.NameOrID = ctrs[0].ID
+	}
+
```

Comment:
```suggestion
```
We don't support `latest` on remote commands.  The use case is, n-people sharing a podman instance so what does `latest` mean?